lib/filter.js

```diff
@@ -11,12 +11,45 @@ exports.parseFilter = (filter) => {
     throw new Error('Invalid filter operator');
   }
 
+  if (filter.hasOwnProperty('matchMissing') && !(typeof filter.matchMissing === 'boolean')) {
```

Comment:
I tried to find a way to do that. Any suggestions? Given that we can do it for any possible input such as `'hello'`, `'true'`, `1`, `undefined` etc